wsgi/odyssy/basic/views.py

```diff
@@ -1,5 +1,11 @@
 from django.shortcuts import render
 
+from announcement import views as announcement_view
 
 def index(request):
-    return render(request, 'basic/index.html')
+    """ Index page of the Website """
+    latest_announcement = announcement_view.latest_announcement(5)
```

Comment:
You have to display list of all active announcement. So, here you have to apply a filter which filters out all announcements where end_date is less than current date.